apps/sms/test/marionette/lib/messages.js

```diff
@@ -9,6 +9,26 @@
     BACKSPACE: '\ue003'
   };
 
+  function observeElementStability(el) {
+    delete el.dataset.__stable;
+
+    function markElementAsStable() {
+      return setTimeout(function() {
+        el.dataset.__stable = 'true';
+        observer.disconnect();
+      }, 1000);
```

Comment:
mmm, there can be several changes (eg. recipients is re-rendered 4 times for a single "add" operation) in a row with unknown time between. Maybe I'm just missing your idea, could you please elaborate more?